xarray/core/dataset.py

```diff
@@ -2720,6 +2848,32 @@ def tail(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Example
+        -------
+
+        # Sample dataset
+        >>> data = xr.DataArray([[1, 2, 3], [4, 5, 6], [7, 8, 9]], dims=("x", "y"))
+        >>> dataset = xr.Dataset({"data": data})
+
+        # Print the original dataset
```

Comment:
I don't think we really need to have a comment saying that we are printing the dataset it time, because once the docstring is compiled that part should be fairly obvious to the reader.

Reply:
Should I remove the comment # Print the original dataset or should I not print the original dataset. I did not actually understand what you meant to say